src/ast/mod.rs

```diff
@@ -1004,6 +1005,51 @@ pub enum OneOrManyWithParens<T> {
     Many(Vec<T>),
 }
 
+impl<T> Deref for OneOrManyWithParens<T> {
```

Comment:
This is looking good -- I think if we are going to add this code we should also add examples to the documentation f `OneOrManyWithParens` otherwise it will likely be hard to find

So like 
```rust 
/// # Example: accessing as a slice:
/// ...
```

```rust 
/// # Example: iterating
/// ...
```

Reply:
Done!